README.md

```diff
@@ -290,6 +291,77 @@ See the [Open Projects](#open-projects) section for a detailed description of th
 See the [Settings](#i18n-settings) section of the [Internationalization (i18n)](#i18n) paragraph for a detailed description of these parameters.
 
 
+<a name="virtual-machine-instructions"></a>
+### Virtual Machine Instructtions
+
+These instructions allow to deploy a testing or development instance of Fab Manager inside a virtual
+machine, with most of the software dependencies installed automatically and avoiding to install a lot
+of software and services directly on the host computer.
+
+**Note:** The provision scripts configure the sofware dependencies to play nice with each other while
+they are inside the same virtual environment but said configuration is not optimized for a production
+environment.
+
+1. Install [Vagrant][vagrant] and [Virtual Box][virtualbox] (with the extension package).
+
+2. Retrieve the project from Git
+
+   ```bash
+   git clone https://github.com/MakerNetwork/MakerNet.work.git
+   ```
+
+3. From the project directory, run:
+
+   ```bash
+   vagrant up
+   ```
+
+4. Once the virtual machine finished building, reload it with:
+
+   ```bash
+   vagrant reload
+   ```
+
+5. Log into the virtual machine with:
+
+   ```bash
+   vagrant ssh
+   ```
+
+6. While logged in, navigate to the project folder and install the Gemfile
+   dependencies:
+
+   ```bash
+   cd /vagrant
+   bundle install
+   ```
+
+7. Set a directory for Sidekick pids:
+
+   ```bash
+   mkdir -p tmp/pids
+   ```
+
+8. Copy the default configuration files:
+
+   ```bash
+   cp config/database.yml.virtual config/database.yml
+   cp config/application.yml.default config/application.yml
+   ```
+
+10. Set up the databases. (You should provide the desired admin credentials):
+
+   ```bash
+   ADMIN_EMAIL='youradminemail' ADMIN_PASSWORD='youradminpassword' rake db:setup
```

Comment:
As stated at point 8 of [Setup a development environment](https://github.com/LaCasemate/fab-manager/blob/master/README.md#general-guidelines) procedure, `rake db:setup` will not propagate the raw SQL instructions needed for members autocompletion and other features.

Reply:
Yikes! I had not seen that before. It has the complete instructions and a warning legend now.